src/extensions/default/Autosave/main.js

```diff
@@ -51,12 +51,11 @@ define(function (require, exports, module) {
             doc.releaseRef();
             clearTimeout(pending[path]);
             delete pending[path];
-			
-			// if statement cheks if there is no more pending saves
-			if(Object.keys(pending).length === 0) {
-				// call triggerProjectDirty to handles dirtyFlagChange event
-				BrambleEvents.triggerProjectSaved(path);
-			}
+
+        // if statement cheks if there is no more pending saves
+        if(Object.keys(pending).length === 0) {
+            BrambleEvents.triggerProjectSaved(path);
+        }
```

Comment:
two `}` on successive lines with the same indentation is a dead giveaway that indents went terribly wrong: it looks like L59 through L74 are all indented to far: can you confirm whether this is the case?